scripts/generate-documentation.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+#Generate documentation for the project based on Typescript types
+pnpm exec typedoc
+
+# Typedoc generates a README.md file in the docs folder, which is not needed
```

Comment:
I think the comment is explicit here,
I couldn't find a way to avoid `typedoc` to regenerate `README.md` file in `docs` folder